src/components/ArticleCard.jsx

```diff
@@ -0,0 +1,14 @@
+import { shortenTitle } from "../utils/text";
+
+function ArticleCard({ article }) {
+    const { title, article_img_url } = article;
+
+    return (
+        <section className="article-card">
+            <img src={article_img_url} alt={`article titled ${title}`} />
+            <div>{shortenTitle(title)}</div>
+        </section>
```

Comment:
This doesn't show the user an awful lot of info - we have more data available, so consider making use of it and telling the user a bit more. (author, date ect)